### DISCUSSION TITLE
GitHub Student Pack

### DISCUSSION THREAD

**Discussion by partfix** [community/community#63702] [GitHub Education]
### Select Topic Area

GitHub Education 

### Body

Hello good day, 
 
  Can I ask how many days or week to get the GitHub Benefits?  The semester are days away and I just realized How GitHub Student pack helpful for me as a University student. I hope someone from GitHub, or anyone can answer my concern.
  
  
<img src="https://github.com/community/community/assets/114133634/3ce537d1-c849-466a-ac52-71e1eee361bf" height= "330px">

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @rence-scratch ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by AshishKumar9532** [comment]
Ig they'll take around 72 hours to release the benefits into your account,I'm also in the waiting list my account was approved at 12th of the August but no benefits.

**Reply by partfix** [reply]
I see I had mine last august 11 and some of another applicant have an issue too regarding on it for a week some of them months.  Thank you for answering my question!! really appreciate it.
